spec/android_spec.rb

```diff
@@ -190,14 +190,14 @@
           out = "package: name='bbc.iplayer.android' yyyyy='xxxxxxxx' qqqqq='rrrrrrrr'"
 
           device = DeviceAPI::Android::Device.new(serial: 'SH34RW905290')
-          allow(Open3).to receive(:capture3) { [out, '', $STATUS_ZERO] }
+          allow(Open3).to receive(:capture3) { [out, '', STATUS_OK] }
           expect { device.app_version_number('iplayer.apk') }.
               to raise_error(StandardError, 'Version number not found')
         end
 
         it 'can raise an error if aapt can not be found' do
           device = DeviceAPI::Android::Device.new(serial: 'SH34RW905290')
-          allow(Open3).to receive(:capture3) { ['', '', $STATUS_ONE] }
+          allow(Open3).to receive(:capture3) { ['', '', STATUS_ERROR] }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.